CMSgov/mdct-mcr
# MDCT-414: Parser Edition

## Description


Fixes [MDCT-414](https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-414)

![Screen Shot 2022-08-02 at 10 21 36 AM](https://user-images.githubusercontent.com/19439679/182397842-9c5bfd41-1fc7-4891-8927-ad85c416f369.png)

### How to test

1. Go to /mcpar/state-level-indicators/program-characteristics
2. Check out the page!

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
